pcsx2/CDVD/CDVD.cpp

```diff
@@ -839,7 +839,7 @@ void cdvdReset()
 
 	// If we are recording, always use the same RTC setting
 	// for games that use the RTC to seed their RNG -- this is very important to be the same everytime!
-#ifndef DISABLE_RECORDING
+#ifndef PCSX2_CORE
```

Comment:
```suggestion
#ifndef PCSX2_CORE
const bool input_recording_active = g_InputRecording.IsActive();
#else
const bool input_recording_active = g_InputRecording.isActive();
#endif
if (input_recording_active)
```
Although IMO just making the naming consistent would be better (all the core stuff follows TitleCase, not camelCase).

Reply:
Yeah we probably need to actually codify a style guide, because that's contrary to what was proposed a while ago https://github.com/tadanokojin/pcsx2/blob/coding-guide/pcsx2/Docs/Coding_Guidelines.md#camel-case-for-function-names which is what i try to follow

Reply:
All the code outside of the Qt classes (because they, themselves, use camelCase) is TitleCase, perhaps the style guide should be adjusted to suit..

Reply:
I believe the code was that way (TitleCase) at the time the style guide was originally drafted as well.  But yes either the style guide is adjusted to fit what the majority of the code base already is, or new code is incrementally written in the desired style.  

Something has to be committed to or there is no hope for consistency (if we care about it)